config.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
```

Comment:
I frankly do not see the point of making ports configurable for K8s charms. One can remap at pod level AND service level! IMO hardcode 9091 and simplify the charm.

Reply:
Yeah I agree, I removed the config for `port` and `grpc-port`.